understanding/20/audio-control.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Control</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>A page that plays music or sounds doesn't disrupt people</dd>
-		<dt>Author task</dt><dd>If you play audio content automatically, let people turn it down or off</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>A page that plays music or sounds doesn't disrupt people.</dd>
+         <dt>What to do</dt><dd>If you play audio content automatically, let people turn it down or off.</dd>
+         <dt>Why it's important</dt><dd>Sound can distract some users or interfere with their assistive technology.</dd> 
```

Comment:
better!

Reply:
I guess we could remove the "can" like this:
> Sound distracts some users or interferes with their assistive technology.